app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if command_parts[0] != "cp" or command_parts[1] == command_parts[2]:
+        print("Command is not 'cp' or source "
```

Comment:
imho u can use variables, like source/destination_file or smth, not with index. I think it will be look better.